Sources/AppDelegate.swift

```diff
@@ -148,12 +148,22 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
     
     private func getAllBalances() {
-        guard let balances = UpbitServices.shared.getBalances() else { return }
-       
-        let currency: [String] = balances.map {  return "\($0.unitCurrency)-\($0.currency)" }
-                                        .filter { false == $0.contains("KRW-KRW") }
-                                        .filter { false == $0.contains("KRW-LUNA") }
-        
+        guard let balances = UpbitServices.shared.getBalances(),
+              let allMarket = UpbitServices.shared.getAllMarket() else { return }
+        
+        let availableMarkets = allMarket.compactMap{ $0.market }
+        let currency: [String] = balances
+            .map { "\($0.unitCurrency)-\($0.currency)" }
+            .filter{ market -> Bool in availableMarkets.filter{ $0 == market }.first != nil }
+        
+        var myCurrency: [String: Market]{
+            var tempDict: [String: Market] = [: ]
```

Comment:
```suggestion
            var tempDict: [String: Market] = [:]
```
불필요한 공백이 들어간 것 같습니다.